docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,32 +1,99 @@
+|31/03/2008|441854|house|2       |
+|30/06/2016|430880|unit |2       |
+|30/09/2016|430654|unit |2       |
++----------+------+-----+--------+
 ```
 
-## Create a time series predictor
+Where:
 
-Now, we can specify that we want to forecast the `MA` column, which is a moving average of the historical median price for house sales. However, looking at the data you can see several entries for the same date, depending on two factors: how many bedrooms the properties had, and whether properties were "houses" or "units". This means that we can have up to ten different groupings here (although, if you do some digging, you will find we only actually have seven of the possible ten combinations in practice).
+| Column                | Description                                                                                  | Data Type           | Usage   |
+| :-------------------- | :------------------------------------------------------------------------------------------- | ------------------- | ------- |
+| `saledate`            | The date of sale                                                                             | `date`              | Feature |
+| `MA`                  | Moving average of the historical median price of the house or unit                           | `integer`           | Label   |
+| `type`                | Type of property (either a house or a unit)                                                  | `character varying` | Feature |
+| `bedrooms`            | Number of bedrooms                                                                           | `integer`           | Feature |
 
-MindsDB makes it simple so that we don't need to repeat the predictor creation process for every group there is. Instead, we can just group for both columns and the predictor will learn from all series and enable forecasts for all of them! The command for this is:
+!!!Info "Labels and Features"
+
+    A **label** is the thing we're predicting — the y variable in simple linear regression.
+    A **feature** is an input variable — the x variable in simple linear regression.
+
+## Training a Predictor Via [`#!sql CREATE PREDICTOR`](/sql/create/predictor)
+
+Now, we can specify that we want to forecast the `MA` column, which is a moving average of the historical median price of the house or unit. However, looking at the data you can see several entries for the same date. It depends on two factors: 1. how many bedrooms the properties have, and 2. whether properties are "houses" or "units". This means that we can have up to ten different groupings here. Although, if you do some digging, you will find we only have seven of the possible ten combinations.
+
+MindsDB makes it simple so that we don't need to repeat the predictor creation process for every group there is. Instead, we can just group for both columns and the predictor will learn from all series and enable forecasts for all of them! Here is the SQL command to do so:
 
 ```
```

Comment:
Missing sql flavour on code block.

Reply:
Could you please give me some details on that?